### PR TITLE
Removed cryo, replaced it with a shower room.

### PR DESCRIPTION
**Pull request by Buggy123**
Cryo is essentially useless anyway, even when completely naked it takes a extremely long time for cryoxadone to start affecting you.

In many cases, it's actually faster to get into cryo, wait a bit to get cryo chems, then jump out and run to a cold shower or even escape and jump out of the airlock.

This PR takes this to it's logical conclusion and replaces cryo with a shower room, complete with soap and a wrench.

If you want to heal a patient with cloning damage, get cryoxadone from a chemist and stick them under a cold shower, much faster than those old, high-tech cryo chambers. If you want to heal a patient who doesn't have clone damage, well no one used cryo for that anyway and the sleepers are still there.

[Picture](http://i.imgur.com/AWRchxO.jpg)

### CODE REVIEW

**Comment by PKPenguin321** [comment]
KEK

**Comment by optimumtact** [comment]
Perfect :+1:

**Comment by bgobandit** [comment]
no

**Comment by Buggy123** [comment]
This is only half meme I honestly want this merged. It's a improvement.

**Comment by MrPerson** [comment]
toppest of keks

**Comment by PKPenguin321** [comment]
this is a legitimate buff too that's the best part

**Comment by Buggy123** [comment]
Can we test merge this? I want to know what people think.

**Comment by MMMiracles** [comment]
you know what kid.

**Comment by Buggy123** [comment]
PK I honestly considered just replacing that entire area with space but I HONESTLY want this merged so I decided against going full meme.

**Comment by Buggy123** [comment]
Compiling and preparing screenshots now.

**Comment by newfren** [comment]
:+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1:

**Comment by Buggy123** [comment]
[Picture](http://i.imgur.com/5ZJoUGh.png)

**Comment by PKPenguin321** [comment]
it looks so clean without the ugly pipes and freezer
:+1:

**Comment by Buggy123** [comment]
Weird something is wrong with the showers, they should be sitting against the wall.

**Comment by MMMiracles** [comment]
>meme pr
>can't even get it right the first time

6/10 ok effort

**Comment by AdamElTablawy** [comment]
>makes a meme PR
>it's actually effective

jesus fuck

i sort of want this now

**Comment by Buggy123** [comment]
Half meme, it's not a full meme pr. I actually want it merged.

**Comment by Buggy123** [comment]
Fixed the shower direction:

[Picture](http://i.imgur.com/AWRchxO.jpg)

**Comment by Zilenan91** [comment]
Actually does medbay have showers in any other area than cryo normally?

**Comment by Buggy123** [comment]
Nope, which is why this makes sense. Meta has a shower but not box.

**Comment by Zilenan91** [comment]
Should probably remove the memes and make one of hte patient rooms into a shower room.

**Comment by Core0verload** [comment]
Kek. Now open a buff cryo PR to actually fix that shit.

**Comment by Buggy123** [comment]
Are you kidding? Coderbus HATES cryo, It would never be merged.

**Comment by xxalpha** [comment]
Why not make it a cold room?

**Comment by Buggy123** [comment]
I could do some sort of actual cryo-chamber I suppose.

**Comment by Cheridan** [comment]
@Buggy123 stop with the generalizations

**Comment by Buggy123** [comment]
At this rate I should just remove medbay.

**Comment by Zilenan91** [comment]
why back in my day spessmen went to the asteroid, killed hivelords with their bare hands, and ate the remains with nothing but their naked, shitty bodies and a bag of chips

**Comment by PKPenguin321** [comment]
>obviously we should nerf cold showers tho

Nnnnnooooooooooooooooo

**Comment by Bawhoppen** [comment]
merge this so someone fixes this shit sometime :+1:

**Comment by Core0verload** [comment]
>obviously we should nerf cold showers tho

NO-NO-NO-NO-NO

**Comment by Steelpoint** [comment]
This is the kind of PR that's meant to get someone more knowledgeable to fix the issue, only for this to be merged and the issue forgotten about.

**Comment by Xhuis** [comment]
I was preparing to write out a long, verbose rant about you and this PR, but then I realized that it's actually a legitimate concern. Showers _are_ better at chilling you than cryo.

I still think that this is a silly change. This is a high-tech space station, and the medical facilities should be more advanced than injecting someone with medicine and stuffing them under a shower. This would also make multiple people cryo-able in one tile, which is a big no-no. Cryo is meant as an efficient, but slow healing method, costing almost nothing but taking a decent amount of time. It's effectiveness varies on the coolness of the air that the cooler can reach - have someone upgrade the parts, and cryo is now better.

Long story short, jury-rigged health showers in a well-funded medical facility are silly and shouldn't exist.

**Comment by MMMiracles** [comment]
Just leave a note saying "sorry, budget cuts" and it'll be fine.

**Comment by Xhuis** [comment]
I hear that argument a lot and it's baseless. "low budget lol" doesn't work on a station with a fully sentient artificial intelligence that's obviously important enough to be targeted by a lot of corporate enemies.

**Comment by Steelpoint** [comment]
Perhaps NT values their highly expensive artificial intelligence to a bunch of cryrogenic machines that are out of date and no longer function properly.

If a cold shower has a better effect than a expensive piece of machinery, why waste money maintaining it when you can just get the cold shower?

But seriously this just highlights needing someone to buff cyro.

**Comment by xxalpha** [comment]
So what is the problem, cryo doesn't cool you down fast enough?

**Comment by Shapsy** [comment]
This actually would end up being a nerf funnily enough since cryo cells multiply reagents when injecting people, so you'd run out of cryoxadone after a couple of patients with this

**Comment by drynwyn** [comment]
oh no, you would run out of the thing that anyone in medbay with access to a fire extinguisher and the wiki can effortlessly shit out ten billion units of

**Comment by YotaXP** [comment]
What if we just made it an actual cold room, like what telecomms used to be, only with breathable air?

**Comment by X-TheDark** [comment]
Why just not make Cryo Cells innately cold rather than requiring Freezer and cooling time?

**Comment by Buggy123** [comment]
Fixed the merge conflict. Blame the 7000 changed lines on mapmerge being broken.

**Comment by optimumtact** [comment]
Not broken, converting your map to a better format.

**Comment by duncathan** [comment]
Actually oranges I think it's converting it to be worse
There's like 65000 lines removed it appears this map is not in tgm format

**Comment by optimumtact** [comment]
Looks like you're right, perhaps he's still using the old map merger on the new map? anyway this is a meme pr so it doesn't matter :3

**Comment by Buggy123** [comment]
Fuck no i want this merged. The new map merge tool just gives me a python error, guess i should try and fix it.

**Comment by Bawhoppen** [comment]
if you think this is actually getting merged you'll be in for a surprise

**Comment by Buggy123** [comment]
Oh joy I love surprises!

**Comment by Shadowlight213** [comment]
Merge when?

**Comment by xxalpha** [comment]
When the map is in the proper format.

**Comment by Buggy123** [comment]
Working on convoluted solution, should be fixed today assuming map merge isn't OS sensitive.

**Comment by Buggy123** [comment]
Map merge's dmm mode is broken and none of the fixes for it work, so its either converting this to tgm or no mapmerging at all.

**Comment by Cheridan** [comment]
ok

**Comment by Buggy123** [comment]
Aight, i'll message you to reopen if i can fix it.
